walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -20,118 +6,45 @@
 import java.util.List;
 import java.util.Set;
 
+/*
+  Assumptions: the collection of events is in sorted order by start time
```

Comment:
Let's call these preconditions and put them on the methods that require this state.

Reply:
Done